docs/examples/ex_extract.rst

```diff
@@ -0,0 +1,456 @@
+
+The `Coordinates` `time`, `latitude`, `longitude`, and `altitude` are all
+one-dimensional and directly relevant to a physical satellite location. The
+equivalent satellite variables are `latitude`, `longitude`, and `altitude`,
+with `time` taken from the Instrument's associated datetimeindex
+(`inst.data.index`).
+
+.. code:: python
+
+   In []: inst.variables
+   Out[]:
+   Index(['uts', 'mlt', 'slt', 'longitude', 'latitude', 'altitude', 'orbit_num',
+          'dummy1', 'dummy2', 'dummy3', 'dummy4', 'string_dummy', 'unicode_dummy',
+          'int8_dummy', 'int16_dummy', 'int32_dummy', 'int64_dummy',
+          'model_dummy2'],
+         dtype='object')
+
+   In []: inst.data.index
+   Out[]:
+   DatetimeIndex(['2009-01-01 00:00:00', '2009-01-01 00:00:01',
+                  '2009-01-01 00:00:02', '2009-01-01 00:00:03',
+                  '2009-01-01 00:00:04', '2009-01-01 00:00:05',
+                  '2009-01-01 00:00:06', '2009-01-01 00:00:07',
+                  '2009-01-01 00:00:08', '2009-01-01 00:00:09',
+                  ...
+                  '2009-01-01 23:59:50', '2009-01-01 23:59:51',
+                  '2009-01-01 23:59:52', '2009-01-01 23:59:53',
+                  '2009-01-01 23:59:54', '2009-01-01 23:59:55',
+                  '2009-01-01 23:59:56', '2009-01-01 23:59:57',
+                  '2009-01-01 23:59:58', '2009-01-01 23:59:59'],
+                 dtype='datetime64[ns]', name='Epoch', length=86400, freq=None)
+
+
+Interpolating `model` data onto `inst` is accomplished via
+
+.. code:: python
+
+   new_data_keys = pysatModels.utils.extract.instrument_view_through_model(inst,
+                              model.data, ['latitude', 'longitude', 'altitude'],
+                              ['latitude', 'longitude', 'altitude'], 'time',
+                              'time', ['deg', 'deg', 'km'], ['dummy2'])
+
+where `inst` and `model.data` provide the required :py:class:`pysat.Instrument`
+object and :py:class:`xarray.Dataset`. The ::
+
+   ['latitude', 'longitude', 'altitude']
+
+term provides the content and ordering of the coordinates for model variables
+to be interpolated. The subsequent ::
+
+   ['latitude', 'longitude', 'altitude']
+
+term provides the equivalent content from the satellite's data set, in the same
+order as the model coordinates. In this case, the same lables are used for
+both the satellite and modeled data sets. The ::
+
+   'time', 'time'
+
+terms cover the model labels used for time variable and coordinate. The ::
+
+   ['deg', 'deg', 'km']
+
+term covers the units for the model dimensions (latitude/longitude/altitude).
+Units for the corresponding information from `inst` are taken directly from the
+:py:class:`pysat.Instrument` object. The final presented input::
+
+    ['dummy2']
+
+is a list of model variables that will be interpolated onto `inst`.
+
+The results of ::
+
+    inst[new_data_keys].plot(title='Interpolation Example')
+
+are shown below.
```

Comment:
There is no figure I could find.  Also, this figure is hard to read.  Recommend making a more informative plot that illustrates an aspect of the extraction process.